### ISSUE TITLE
Accidentally got `ros console list` to show a current console that's not actually running

### ISSUE THREAD

**Issue by deniseschannon** [rancher/os#1225]
**RancherOS Version: (ros os version)**
v0.6.1

Accidentally typed `sudo console list`, which made the enabled console the current console in `sudo ros console list`. 

```
$ sudo ros console enable ubuntu
Pulling console (rancher/os-ubuntuconsole:v0.6.1)...
v0.6.1: Pulling from rancher/os-ubuntuconsole
952132ac251a: Pull complete 
82659f8f1b76: Pull complete 
c19118ca682d: Pull complete 
8296858250fe: Pull complete 
24e0251a0e2c: Pull complete 
759e1014557b: Pull complete 
7b25038eff72: Pull complete 
4f731e4aa7b1: Pull complete 
e12e4ab859a9: Pull complete 
407c83262497: Pull complete 
3f7dc5baf3d2: Pull complete 
Digest: sha256:cb32b3a85cb1b07e38987a5f9de8bf8b5578ee37d1209dad9c45670706dcd464
Status: Downloaded newer image for rancher/os-ubuntuconsole:v0.6.1
$ sudo ros console list
disabled centos
disabled debian
current  default
disabled fedora
enabled  ubuntu
disabled xfce
# Accidentally typed this command
$ sudo console list
ERRO[0000] /usr/sbin/sshd -D : exit status 255          
ERRO[0000] /usr/sbin/sshd -D : exit status 255          
ERRO[0000] /usr/sbin/sshd -D : exit status 255          
ERRO[0000] /usr/sbin/sshd -D : exit status 255          
ERRO[0000] /usr/sbin/sshd -D : exit status 255          
ERRO[0000] /usr/sbin/sshd -D : exit status 255          
ERRO[0000] /usr/sbin/sshd -D : exit status 255          
ERRO[0000] /usr/sbin/sshd -D : exit status 255          
ERRO[0000] /usr/sbin/sshd -D : exit status 255          
ERRO[0000] /usr/sbin/sshd -D : exit status 255          
ERRO[0000] /usr/sbin/sshd -D : exit status 255          
ERRO[0000] /usr/sbin/sshd -D : restarted too fast, not executing 
^C
$ sudo ros console list 
disabled centos
disabled debian
disabled default
disabled fedora
current  ubuntu
disabled xfce
$ sudo system-docker ps
CONTAINER ID        IMAGE                       COMMAND                  CREATED             STATUS              PORTS               NAMES
71a59610f262        nginx                       "nginx -g 'daemon off"   3 minutes ago       Up 3 minutes                            grave_colden
17c766201732        rancher/os-docker:1.12.1    "/usr/bin/user-docker"   53 minutes ago      Up 53 minutes                           docker
819be88ac952        rancher/os-console:v0.6.1   "/usr/bin/ros entrypo"   53 minutes ago      Up 53 minutes                           console
02f6129d2f30        rancher/os-base:v0.6.1      "/usr/bin/ros entrypo"   53 minutes ago      Up 53 minutes                           network
f2479f2077e8        rancher/os-base:v0.6.1      "/usr/bin/ros entrypo"   54 minutes ago      Up 54 minutes                           ntp
8c6e9f6f1ed8        rancher/os-udev:v0.6.1      "/usr/bin/ros entrypo"   54 minutes ago      Up 54 minutes                           udev
b10099c79358        rancher/os-syslog:v0.6.1    "/usr/bin/ros entrypo"   54 minutes ago      Up 54 minutes                           syslog
d8c1ac94d721        rancher/os-acpid:v0.6.1     "/usr/bin/ros entrypo"   54 minutes ago      Up 54 minutes                           acpid
```

**Comment by ibuildthecloud** [comment]
@joshwget Does `console` really need to be a user accessible thing?  Couldn't it be hidden in various ways.  Hidden subcommand or don't symlink it by default, etc?

**Comment by joshwget** [comment]
@ibuildthecloud No, it should be a hidden subcommand.

**Comment by joshwget** [comment]
This will be fixed by #1279 and #1280. Closing in favor of those.
